### DISCUSSION TITLE
Exposing additional parameters of Counter

### DISCUSSION THREAD

**Discussion by rsiatka** [abstracta/jmeter-java-dsl#229] [Ideas] [answered]
I can see that JMeter DSL allows to use counter config element. Unfortunately, it only provides the ability to define the variable name and initial value. In my case i need to select additional option "Track counter independently for each  user" which is disabled by default.
It would be great if there were additional counter options, such as:
- Increment
- Maximum value
- Track counter independently for each  user

<img width="353" alt="image" src="https://github.com/abstracta/jmeter-java-dsl/assets/45803776/de67d235-967c-45e5-85d2-4752d8950ef8">

**Comment by rabelenda** [comment, marked as answer]
Sure! Thank you for asking this! 

It should be quite easy to do if you would like to submit a PR. Otherwise we could implement them in the following days.

**Reply by rabelenda** [reply]
We have just released a [new version](https://github.com/abstracta/jmeter-java-dsl/releases/tag/v1.22) which includes the new settings for counters.

**Reply by rsiatka** [reply]
Thank you @rabelenda. This feature will reduce my testing time tenfold.
